lcc_members/__init__.py

```diff
@@ -1,6 +1,17 @@
 # Copyright 2020 Lokavaluto ()
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-
 from . import models
 from . import controllers
 from . import wizard
+from odoo.api import Environment, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
```

Comment:
I am not sure it is a post-init_hook that is needed. At module installation time, no partner is "Main partner" as this field is added with the module update. Then the research can't match any partner, can it?
What about a planned action? or a button to activate the public profile creation process?